app/models/miq_queue.rb

```diff
@@ -153,6 +153,19 @@ def self.put(options)
     msg
   end
 
+  # Execute a job on all servers.  Filterable by zone
```

Comment:
I think the method will be self-explanatory if the parameter is `zone` rather than an options hash.

Reply:
What's the use case for broadcasting to all servers in a zone?  What about roles?  The existing embedded ansible is 1 regional role.

I saw this in the linked RFE issue but it's still unclear:
> Verify multi-role support (incl. zones and broadcast of deletes)

Either way, we should make broadcast as simple as possible.  Being able to filter who the recipients are goes against the name "broadcast".  The method could know if it's running on a server that should run it.  I know it's wasteful but for every infrequent activities, broadcasting a job to all servers with the queued method knowing if it should run the method could make more sense for the term "broadcast".